packages/frontend/app/(dashboard)/course/[cid]/queue/[qid]/components/TAQuestionCardButtons.tsx

```diff
@@ -259,45 +268,94 @@ const TAQuestionCardButtons: React.FC<TAQuestionCardButtonsProps> = ({
-            question.isTaskQuestion
-              ? tasksSelectedForMarking.length > 0
-                ? 'Mark ' + tasksSelectedForMarking.join(', ') + ' as Done'
-                : 'Mark All as Done'
-              : 'Finish Helping'
-          }
-          open={isFinishHelpingTooltipVisible}
-        >
-          <CircleButton
-            onMouseEnter={() => setIsFinishHelpingTooltipVisible(true)}
-            onMouseLeave={() => setIsFinishHelpingTooltipVisible(false)}
-            variant="green"
-            icon={<CheckOutlined />}
-            loading={finishHelpingButtonLoading}
-            disabled={cantFindButtonLoading || requeueButtonLoading}
-            onClick={() => {
-              // setCheckOutTimer()
-              setFinishHelpingButtonLoading(true)
-              if (
-                question.isTaskQuestion &&
-                tasksSelectedForMarking.length > 0
-              ) {
-                markSelected().then(() => {
-                  setFinishHelpingButtonLoading(false)
-                })
-              } else {
-                changeStatus(ClosedQuestionStatus.Resolved).then(() => {
-                  setFinishHelpingButtonLoading(false)
-                })
+        {question.status !== OpenQuestionStatus.Paused && (
+          <Tooltip
+            title={
+              question.isTaskQuestion
+                ? tasksSelectedForMarking.length > 0
+                  ? 'Mark ' + tasksSelectedForMarking.join(', ') + ' as Done'
+                  : 'Mark All as Done'
+                : 'Finish Helping'
+            }
+            open={isFinishHelpingTooltipVisible}
+          >
+            <CircleButton
+              onMouseEnter={() => setIsFinishHelpingTooltipVisible(true)}
+              onMouseLeave={() => setIsFinishHelpingTooltipVisible(false)}
+              variant="green"
+              icon={<CheckOutlined />}
+              loading={finishHelpingButtonLoading}
+              disabled={cantFindButtonLoading || requeueButtonLoading}
+              onClick={() => {
+                // setCheckOutTimer()
+                setFinishHelpingButtonLoading(true)
+                if (
+                  question.isTaskQuestion &&
+                  tasksSelectedForMarking.length > 0
+                ) {
+                  markSelected().then(() => {
+                    setFinishHelpingButtonLoading(false)
+                  })
+                } else {
+                  changeStatus(ClosedQuestionStatus.Resolved).then(() => {
+                    setFinishHelpingButtonLoading(false)
+                  })
+                }
+              }}
+            />
+          </Tooltip>
+        )}
+        {question.status === OpenQuestionStatus.Paused ? (
+          <div className="relative ml-3 inline-flex items-center justify-center">
+            <div className="absolute inset-0 animate-ping rounded-[50%] bg-green-300 opacity-50 before:content-['']"></div>
+            <Tooltip title="Resume Helping">
```

Comment:
the pulse is too much imo. I think the yellow highlight with it being at the top of the queue is enough to make sure it's not forgotten lol